blocks/identity-block/components/FormInputField/styles.scss

```diff
@@ -0,0 +1,47 @@
+.xpmedia-form-field {
+  color: $ui-primary-font-color;
+  display: block;
+  font-size: calculateRem(20px);
+  line-height: 2.5;
+  margin: map-get($spacers, 'md') 0;
+}
+
+.xpmedia-form-field-input,
+.xpmedia-form-field-label,
+.xpmedia-form-field-tip {
+  font-size: calculateRem(16px);
+  line-height: 1.5;
+  margin: map-get($spacers, 'xxs') 0;
+}
+
+.xpmedia-form-field-label {
+  font-weight: bold;
+}
+
+.xpmedia-form-field-input {
+  border: solid 1px $border-rule-color;
+  border-radius: calculateRem(4px);
+  display: block;
+  outline: none;
+  padding: map-get($spacers, 'xs');
+  width: 100%;
+
+  &--error {
+    border-color: $danger-color;
+    color: $danger-color;
+  }
+}
+
+.xpmedia-form-field-tip {
+  color: $ui-medium-gray;
+
+  > span {
+    &::after {
+      content: ' ';
```

Comment:
What's the need to add a blank space after the tip?

Reply:
Word spacing.  To separate the messages as if they were one string of text.

Reply:
I changed this to just add a space to the error message.